xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -348,3 +371,38 @@ void CAndroidUtils::OnSettingChanged(const std::shared_ptr<const CSetting>& sett
   if (settingId == CAndroidUtils::SETTING_LIMITGUI)
     CDisplaySettings::GetInstance().ClearCalibrations();
 }
+
+const std::vector<int> CAndroidUtils::GetDisplaySupportedHdrTypes()
+{
+  CJNIWindow window = CXBMCApp::getWindow();
+
+  if (window)
+  {
+    CJNIView view = window.getDecorView();
+    if (view)
+    {
+      CJNIDisplay display = view.getDisplay();
+      if (display)
+      {
+        CJNIDisplayHdrCapabilities caps = display.getHdrCapabilities();
+        if (caps)
+          return caps.getSupportedHdrTypes();
+      }
+    }
+  }
+
+  return {};
+}
+
+const void CAndroidUtils::LogDisplaySupportedHdrTypes()
+{
+  const std::vector<int> hdrTypes = GetDisplaySupportedHdrTypes();
+  std::string text;
+
+  for (int type : hdrTypes)
+  {
+    text += HdrTypeString(type) + " ";
+  }
+
+  CLog::Log(LOGDEBUG, "CAndroidUtils: Display supported HDR types: {}", text);
```

Comment:
there is potential for improvement here by moving the space to the beginning:
`text += " " + HdrTypeString(type);`
and:
`CLog::Log(LOGDEBUG, "CAndroidUtils: Display supported HDR types:{}", text);`
then the space at the end is now removed

Reply:
can you explain what happens when nothing is supported? is an empty string?

Reply:
> can you explain what happens when nothing is supported? is an empty string?

Yes that's what happens.

Reply:
i think for a better understanding of the log is better add a ternary op. instead pass directly "text"var where when the "text" is empty then pass " None" string

Reply:
I'm not sure why but I can't get a localized string at the moment of the log. It's always empty.

Reply:
usually in the log we should keep english language, not localized language, you can add directly " None"